mkrelease

```diff
@@ -40,7 +40,7 @@ pack()
 	tar -xf "breathe-$version.tar.gz"
 
 	cd -- "breathe-$version"
-	python3 setup.py sdist bdist_wheel
+	python3 -m build
```

Comment:
What does this run? Is there a module called build somewhere?

Reply:
it's not a built-in, but it is a high-profile package by PyPA .. see [docs](https://pypa-build.readthedocs.io/en/latest/)

Reply:
How does it get into our project? I can't see it listed as a dependency? Does it come from `build-backend = "setuptools.build_meta"`?

Reply:
it is implied that `build` is installed beforehand (e.g. [here](https://packaging.python.org/en/latest/tutorials/packaging-projects/#generating-distribution-archives)), since it processes the pyproject.toml file.

Reply:
`pip` doesn't create packages, it just installs them.

Reply:
I thought I had a method for building `sdist` independent of `python3 -m build`, but I can't find it. There is `pip wheel` though.